app/src/main/java/com/neo/speaktouch/model/Type.kt

```diff
@@ -63,6 +65,9 @@ enum class Type {
             // View->TextView->EditText
             if (className `is` EditText::class.java) return EDITFIELD
 
+            // View->TextView->CheckedTextView
+            if (className `is` CheckedTextView::class.java) return CHECKEDTEXT
```

Comment:
Good, I saw that it inherits from Checkable. I think we need to support it as well, to cover custom views.

Reply:
This is the special case. checkedTextView is usually list item and should be announced only if checked (selected, marked, ...). If we generalize by `isCheckable` property, state would be spoken for each item, not only one checked which is not intentional behavior for just one checked item in the list view.

Reply:
Good, I wasn't aware of this particular behavior.